mongodump/prepare.go

```diff
@@ -200,29 +200,17 @@ func (dump *MongoDump) outputPath(dbName, colName string) string {
 	// than 255 bytes long. This includes the longest possible file extension: .metadata.json.gz
 	// The new format is <truncated-url-encoded-collection-name>%24<collection-name-hash-base64>
 	// where %24 represents a $ symbol delimiter (e.g. aVeryVery...VeryLongName%24oPpXMQ...).
-	if len(colName) > 238 {
-		colNameTruncated := util.EscapeCollectionName(colName)[:208]
-		delimiter := util.EscapeCollectionName("$")
+	escapedColName := util.EscapeCollectionName(colName)
+	if len(escapedColName) > 238 {
+		colNameTruncated := escapedColName[:208]
 		colNameHashBytes := sha1.Sum([]byte(colName))
```

Comment:
Minor question -- do we want a hash of the regular collection name, or the escaped collection name? Currently it's the hash of just the collection name, but I don't think it really makes a difference.

Reply:
Yeah it doesn't matter as long as we're consistent. Let's do the unescaped collection name.